doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -202,8 +202,8 @@ An over use of host aggregates and availability zones can result in a granular p
 Recommendation: Separation of control zone and execution zone into different security zones
 
 <a name="3.3.4"></a>
-### 3.3.4. Flavour management (i.e. T-Shirt type)
-Content to be developed
+### 3.3.4. Flavor management
+A flavor defines the compute, memory, and storage capacity of nova instances. When instances are spawned, they are mapped to flavors which define the available hardware configuration for them. For simplicity, the flavors can be named according to T-shirt sizes like XS,S,M,L,XL,2XL,3XL and so on. The specifications for these sizes can be defined based on the category of workloads in terms of resource requirements. One such sample specification is listed [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4211-predefined-compute-flavours).
```

Comment:
The text uses may and sample. In my views we need to use stronger language to request that any favours defined in OpenStack will need to map to the pre-defined ones in RM 4.2.1.1 Predefined Compute Flavours.

Reply:
"The specifications for these sizes can be defined based on the category of workloads in terms of resource requirements. One such sample specification is listed" -> "The specifications for these sizes should map to the predefiend compute flavours listed [here]..."